### PR TITLE
GCM Power Spectrum Modification

### PR DESCRIPTION
**Pull request by LenkaNovak**
### Description

Modifies the GCM power spectrum diagnostics to be computed using the full KE, making them directly comparable to the spectra used in the Jablonowski paper on the baroclinic wave test, which CI uses as a benchmark test. 

- [X] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [X] Unit tests are included OR N/A.
- [X] Code is exercised in an integration test OR N/A.
- [X] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,6 @@ src/**/Manifest.toml
 
 # Julia System Images
 *.so
+
+*.sh
+*.out
```
Comment (kpamnany) [inline .gitignore:40]:
```suggestion
```
I think `.gitignore` should contain only generated files.

**Review by kpamnany — APPROVED** [review]
Changes look fine to me apart from my single comment. I think the GCMDriver uses this diagnostics group, I assume you've verified the output. :)

File: .gitignore (hunk 1 above)
Comment (LenkaNovak) [reply .gitignore:40]: Sounds good! I guess it makes life easier for people who use bash scripts to run the CM, but it's not a biggie either way

**Pull request merged**